pages/workspace/add-events-from-inspector.mdx

```diff
@@ -5,33 +5,75 @@ title: 'Add Events and properties from Inspector'
 
 import Link from '../../components/Link';
 
-## Inspector Dashboard
+The Inspector compares the events and properties coming in to the tracking plan and detects if there are any of the incoming missing from the tracking plan. Event and properties can both be added to the tracking plan from the <Link passHref href="/workspace/inspector/inspector-issues-view"><a>Issues view</a></Link> and the <Link passHref href="/workspace/inspector/inspector-events-view"><a>Events view</a></Link>.
+
+## Event not in Tracking Plan
+
+### Issues view
+
+When an event is missing from the tracking plan, the issue details of the issue view will offer you to add it to the tracking plan right from there. Click the "Add to Tracking Plan" button and the event and all it's properties are created.
 
 <center>
   <img
-    src={require('../../images/inspector/dashboard.png')}
+    src={require('../../images/inspector/add-event-from-issue-details.png')}
     width="100%"
-    alt="Image shows how to change property presence between events"
+    alt='Image shows issue details with "Add to tracking plan" highlighted for the event.'
   />
 </center>
 
-The inspector shows you all events and properties it detects, and lets you know whether they are missing from your tracking plan, and gives you the option to add the event or property to your tracking plan directly from the Inspector dashboard.
+### Events view
+
+If the Inspector detects an Event that is not in your tracking plan, it will be highlighted in the events view with a red tracking plan icon next to the name, along with an "Add" button to add it to your tracking plan. The slide-over in the events view also has an option to add the event right from there. Click either button to add the event and all it's properties to the tracking plan.
```

Comment:
```suggestion
If the Inspector detects an event that is not in your tracking plan, it will be highlighted with a red tracking plan icon next to the name, along with an "Add" button to add it to your tracking plan. The events view sidebar also has an option to add the event right from there. Click either button to add the event and all it's properties to the tracking plan.
```